primitives/src/condition.rs

```diff
@@ -121,10 +119,8 @@ impl Migrate for TrustedIssuerOld {
 /// Type of claim requirements that a condition can have
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
 #[derive(Encode, Decode, Clone, PartialEq, Eq, Debug, Migrate)]
-#[migrate_context(Option<crate::CddId>)]
 pub struct Condition {
     /// Type of condition.
-    #[migrate]
     pub condition_type: ConditionType,
     /// Trusted issuers.
     #[migrate(TrustedIssuer)]
```

Comment:
Was it intentional not to remove the `migrate` here?

Reply:
Yes - TrustedIssuers still need to be migrated as part of Alcyone 2.2 > 2.3.